Palindrome/palindrome_string.cpp

```diff
@@ -0,0 +1,47 @@
+#include<iostream>
+#include<string.h> // to use swap()
+using namespace std;
+void string_palindrome(string);

```

Comment:
Do we need this declaration of this function here? If no, please remove it.

Reply:
okay I will do that now